packages/office-ui-fabric-react/src/common/IAccessiblePopupProps.ts

```diff
@@ -1,34 +1,38 @@
+/**
+ * {@docCategory Modal}
+ */
 export interface IAccessiblePopupProps {
   /**
-   * Sets the HTMLElement to focus on when exiting the FocusTrapZone.
-   * @defaultvalue The element.target that triggered the Panel.
+   * Sets the element to focus on when exiting the control's FocusTrapZone.
+   * @defaultvalue The `element.target` that triggered the control opening.
    */
   elementToFocusOnDismiss?: HTMLElement;
 
   /**
-   * Indicates if this dialog will ignore keeping track of HTMLElement that activated the Zone.
+   * If false (the default), the control's FocusTrapZone will restore focus to the element which
```

Comment:
I wonder if `FocusTrapZone` is an implementation detail here that may not necessarily apply and noticed some interface elements that are duplicated in `FocusTrapZone` below. Should this interface be a `FocusTrapZone` type, possibly implemented by `FocusTrapZone` props?

Reply:
I'm not sure of the background on the duplicated interface elements (or of why IAccessiblePopupProps exists), though I agree that it's weird. Could you clarify what you mean by the second sentence of your comment?

Reply:
I'm wondering if it makes more sense to have in `FocusTrapZone.types.ts`:

```tsx
export interface IAccessiblePopupProps { ... };

export interface IFocusTrapZoneProps extends IAccessiblePopupProps, ... { ... };
```

I know it's a bit outside of the scope of this PR, but I wanted to raise the question before we 
 start exporting `IAccessiblePopupProps`. Although I now see `IDialogProps` is already exporting it, so maybe the point is moot.

Reply:
That makes sense. Since I'm already doing this I'd like to fix it the right way.

Reply:
After some discussion we're leaving this as-is for now.